### PR TITLE
Skill Checks - Virtuoso rolls too high

### PR DESCRIPTION
**Pull request by Tinkastrong**
#502 
Bugfix
Virtuoso rolls (skillRank-2)d6, instead of (skillRank)d6

### CODE REVIEW

File: src/module/system/check/skill.js [hunk 1]
```diff
@@ -32,10 +32,13 @@ class PTUSkillCheck extends PTUDiceCheck {
     prepareModifiers() {
         super.prepareModifiers();
 
+        const skillRank = this.actor.system.skills[this.skill]?.value?.total ?? 1;
+        const diceWrtVirtuoso = this.targetOptions.has("skill:virtuoso") ? skillRank-2 : skillRank;
+
```
Comment (dylanpiera) [inline src/module/system/check/skill.js:37]:
```suggestion
```

File: src/module/system/check/skill.js [hunk 2]
```diff
@@ -32,10 +32,13 @@ class PTUSkillCheck extends PTUDiceCheck {
...
         /** @type {PTUDiceModifier[]} */
         const diceModifiers = [
             new PTUDiceModifier({
-                diceNumber: this.actor.system.skills[this.skill]?.value?.total ?? 1,
+                diceNumber: diceWrtVirtuoso,
```
Comment (dylanpiera) [inline src/module/system/check/skill.js:41]:
```suggestion
                diceNumber: Math.clamped(this.actor.system.skills[this.skill]?.value?.total ?? 1, 1, 6)
```

**Review by dylanpiera — APPROVED** [review]
Simpler solution 😉

**Comment by Tinkastrong** [comment]
I intended it to allow people to homebrew their sheets as they see fit, with fully RAW function.

**Pull request merged**